### PR TITLE
[16][MIG] Migrate stock_warehouse_relationship

### PR DESCRIPTION
**Pull request by florian-dacosta**
I did start from https://github.com/OCA/stock-logistics-warehouse/pull/1558 which was stale.

@petrus-v I guess it may interest you
@lmignon maybe since you had made a review on the 14 PR
@bguillot

### CODE REVIEW

File: stock_warehouse_relationship/i18n/stock_warehouse_relationship copy.pot [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Translation of Odoo Server.
```
Comment (petrus-v) [inline stock_warehouse_relationship/i18n/stock_warehouse_relationship copy.pot]: not sure if this file is present in the former PR! but can be removed!

File: stock_warehouse_relationship/tests/test_warehouse_relationship.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# Copyright 2022 Foodles (http://www.foodles.co).
```
Comment (petrus-v) [inline stock_warehouse_relationship/tests/test_warehouse_relationship.py:1]: this file should probably import from the __init__.py file above unless practices changes in version 16.0 (but I belives I've probably forget it in former PR as long I'm using pytest to launch them

**Review by petrus-v — APPROVED** [review]
Thanks taking care of this migration :+1:

File: stock_warehouse_relationship/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "summary": "Technical module to add warehouse_id field on various stock.* models",
+    "version": "16.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "author": "Akretion, Pierre Verkest, Odoo Community Association (OCA)",
+    "maintainers": ["petrus-v"],
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "stock",
```
Comment (petrus-v) [inline stock_warehouse_relationship/__manifest__.py:16]: just notice that this module is based on  `stock_location_warehouse`
```suggestion
        "stock",
        "stock_location_warehouse",
```

Comment (florian-dacosta) [reply stock_warehouse_relationship/__manifest__.py:16]: I don't think  `stock_location_warehouse` is necessary in v16.
`warehouse_id` field on `stock.location` is present in `stock` module.

Comment (petrus-v) [reply stock_warehouse_relationship/__manifest__.py:16]: ok great I haven't deep into it on v16!

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**